Header header
int8[3] dim
float32[] histograms

int64 hist_npoints
float32[] hpoints3d

int64 original_npoints
float32[] origpoints

sensor_msgs/Image image
